charts/chia/1.3.38/templates/deployment.yaml

```diff
@@ -18,6 +18,11 @@ spec:
     spec:
       # FIXME: Let's please remove hostnetwork when upstream hostport issue is sorted out with kube-router
       hostNetwork: {{ include "hostNetworkingEnabled" . }}
+      {{- if hasKey .Values.global "ixChartContext" }}
```

Comment:
I don't recall but i don't think chia even uses the GPU ?
So only those apps should be updated which are going to consume GPU.

Once we have the new common in, all of this can be dynamic of course.